src/__test__/PageRoutes.test.js

```diff
@@ -0,0 +1,34 @@
+import { Provider } from 'react-redux';
```

Comment:
Your app has unit tests for several components of the app and some pure functions in Redux code! Fantastic job 🔥